language_negotiation_lambda.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+exports.handler = (event, context, callback) => {
+    const response = event.Records[0].cf.response;
+    const headers = response.headers;
+    const customHeaderName = 'X-Accept-Language';
+    const acceptLanguage = 'Accept-Language';
+    const supportedLanguages = ['en', 'es'];
+    const defaultLanguage = 'en';
+
+    const getSelectableLangugaes = function(languages){
+        var selectableLanguages = [];
+        languages.forEach(function(language){
+            var language_data = language.split(';');
+
+            var locale = language_data[0];
+            var weight = language_data[1];
+
+            var sanitizedLocale = locale.toLowerCase().substring(0, 2);
+            var sanitizedWeight = 1.0;
```

Comment:
What is the scale for weight? Does a weight of 1.0 indicate the highest or lowest priority?